### PR TITLE
*: update  building rocksdb.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @BusyJay @shenli @queenypingcap

### CODE REVIEW

File: scripts/build_rocksdb.sh [hunk 1]
```diff
@@ -6,25 +6,48 @@ echo "building rocksdb..."
...
+    fi
+    
+    cd rocksdb-rocksdb-4.6.1 
+    make shared_lib 
+    ${SUDO} make install-shared 
+    # guarantee tikv can find rocksdb.
+    ${SUDO} ldconfig
+}
+
 case "$OSTYPE" in
```
Comment (ngaut) [inline scripts/build_rocksdb.sh]: We should handle CentOS automatically.

Comment (siddontang) [inline scripts/build_rocksdb.sh]: Hi @overvenus 
Could you help us support CentOS later?

Comment (overvenus) [inline scripts/build_rocksdb.sh]: OK, I will try : )

**Comment by siddontang** [comment]
PTAL

**Comment by overvenus** [comment]
I did the same thing today :sweat_smile:

**Comment by ngaut** [comment]
PTAL @overvenus @BusyJay

**Comment by overvenus** [comment]
LGTM

**Comment by shenli** [comment]
LGTM
I have tested it on CentOS 7 and it works.

**Pull request merged**